lib/chef-dk/skeletons/code_generator/templates/default/kitchen.yml.erb

```diff
@@ -5,11 +5,9 @@ driver:
 provisioner:
   name: chef_zero
 
-# Uncomment the following verifier to leverage Inspec instead of Busser (the
-# default verifier)
-# verifier:
-#   name: inspec
-#   format: doc
+verifier:
+  name: inspec
+  format: doc
```

Comment:
The feedback consensus was clear that people want doc format output when running Inspec through test kitchen.

That said, changing the kitchen verifier to default to doc format output and making this less verbose would be great. :)